src/setup/org.freedesktop.IBus.Chewing.gschema.xml

```diff
@@ -59,19 +86,32 @@
             </description>
         </key>
         <key name="max-chi-symbol-len" type="i">
+            <range min="11" max="33"/>
```

Comment:
Test failed because in the test we set `max-chi-symbol-len` to 8.

This value actually can be between 0 and 39. Can you update here and `src/IBusChewingProperties.c` to accept the wider range?

Reply:
no problem!

Reply:
So it's intended that we give smaller value range for user from GUI setting widget? :D

Reply:
I guess so. It works at smaller length but might become inconvenient. Phrases longer than 3x used to trigger O(n^2) pathological case so I guess that's why the default were 11 and 33.